test/Array/array_copyWithin.js

```diff
@@ -0,0 +1,17 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Check for regressions on an issue where copyWithin was failing to look up properties from the prototype.
+var a21_proto = {};

```

Comment:
please use our unittest framework for verbose description and asset. 
search for WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
in the test files.

Reply:
Okay, thanks. I saw a few different styles in existing tests, so I didn't know which to follow.